hoodie-utilities/src/main/java/com/uber/hoodie/utilities/deltastreamer/DeltaSync.java

```diff
@@ -245,7 +246,9 @@ private void refreshTimeline() throws IOException {
       if (lastCommit.isPresent()) {
         HoodieCommitMetadata commitMetadata = HoodieCommitMetadata.fromBytes(
             commitTimelineOpt.get().getInstantDetails(lastCommit.get()).get(), HoodieCommitMetadata.class);
-        if (commitMetadata.getMetadata(CHECKPOINT_KEY) != null) {
+        if (cfg.checkpoint != null && !cfg.checkpoint.equals(commitMetadata.getMetadata(CHECKPOINT_RESET_KEY))) {
```

Comment:
what is the purpose of the second check? 
Also should we allow overriding checkpoint even in the case where `lastCommit=empty` ? For e.g you want to specify from which kafka offsets or timestamp you want to start building the table off?

Reply:
The key save the last checkpoint from the cli.
1. need to work with continuous mode.
2. Prevent overriding the checkpoint again  if forget to delete the cli option .

I have updated to allow overriding in case where last commit is empty